Family.dart

```diff
@@ -0,0 +1,40 @@
+class Family {
+
+  String name;
+  int id;
+  Map<Family,bool> parents = {};
+  Map<Family,bool> children = {};
+
+  Family(int id,String name){
+    this.id  = id;
+    this.name = name;
+  }
+
+  void setName(String name){
```

Comment:
use dart getter and setters like[ this](https://www.geeksforgeeks.org/getter-and-setter-methods-in-dart/)